packages/components/src/components/callout/callout.css

```diff
@@ -6,8 +6,8 @@
   --min-width: 6rem;
   --aspect-ratio: 1;
   --rotation: 0deg;
-  --font-standard: var(--telekom-text-style-body-bold);
   --font-small: var(--telekom-text-style-body-bold);
+  --font-medium: var(--telekom-text-style-body-bold);
```

Comment:
is it needed to have these 2 vars when they reference the same token?

Reply:
not really needed but I think it makes sense, at some point one would want to make medium bigger…